nixos/modules/system/boot/systemd/tmpfiles.nix

```diff
@@ -100,5 +100,22 @@ in
         '';
       })
     ];
+
+    systemd.tmpfiles.rules = [
+      "d  /etc/nixos                         0755 root root - -"
+      "d  /nix/var                           0755 root root - -"
+      "L+ /nix/var/nix/gcroots/booted-system 0755 root root - /run/booted-system"
+      "d  /run/lock                          0755 root root - -"
+      "d  /var/db                            0755 root root - -"
+      "L  /etc/mtab                          -    -    -    - ../proc/mounts"
+      "L  /var/lock                          -    -    -    - ../run/lock"
+      # Boot-time cleanup
+      "R! /etc/group.lock                    -    -    -    - -"
+      "R! /etc/passwd.lock                   -    -    -    - -"
+      "R! /etc/shadow.lock                   -    -    -    - -"
+      "R! /etc/mtab*                         -    -    -    - -"
+      "R! /nix/var/nix/gcroots/tmp           -    -    -    - -"
+      "R! /nix/var/nix/temproots             -    -    -    - -"
```

Comment:
These will only run if `systemd-tmpfiles` is invoked with `--boot` option. Where do we do this?

Reply:
```
$ systemctl cat systemd-tmpfiles-setup.service | grep ^ExecStart=
ExecStart=systemd-tmpfiles --create --remove --boot --exclude-prefix=/dev
```

Reply:
Okay, and that unit seems to have something preventing it from running twice, even if our activation script would get it right. Brr.

Reply:
It's not restarted because we skip units with `RefuseManualStop`